packages/components/src/components/flyout-navigation/flyout-navigation/flyout-navigation.tsx

```diff
@@ -92,6 +91,11 @@ export class FlyoutNavigation {
     setScrollLock(false);
   }
 
+  public componentDidRender(): void {
+    // showModal needs to be called after render cycle to prepare visibility states of dialog in order to focus the dismiss button correctly
```

Comment:
Bad copy & paste with modal references?

Reply:
showModal is the name of the function of the native dialog